bench/src/main/scala/bench/GenBench.scala

```diff
@@ -0,0 +1,104 @@
+package bench
```

Comment:
I don't mind if the package scope was org.scalacheck so there would be _some_ parity with the source.